source/go-perun/new_tutorial/client/handle.rst

```diff
@@ -0,0 +1,138 @@
+            if lcp.NumPeers() != 2 { //TODO:go-perun rename NumPeers to NumParts, Peers to Participants anywhere where all parties are referred to
+                return nil, fmt.Errorf("Invalid number of participants: %d", lcp.NumPeers())
+            }
+
+            // Check that the channel has the expected assets.
+            err := channel.AssetsAssertEqual(lcp.InitBals.Assets, []channel.Asset{c.currency})
+            if err != nil {
+                return nil, fmt.Errorf("Invalid assets: %v\n", err)
+            }
+
+            // Check that we do not need to fund anything.
+            zeroBal := big.NewInt(0)
+            for _, bals := range lcp.FundingAgreement {
+                bal := bals[receiverIdx]
+                if bal.Cmp(zeroBal) != 0 {
+                    return nil, fmt.Errorf("Invalid funding balance: %v", bal)
+                }
+            }
+            return lcp, nil
+        }()
+        if err != nil {
+            r.Reject(context.TODO(), err.Error()) //nolint:errcheck // It's OK if rejection fails.
+        }
+
+To accept the Channel, we follow two steps:
+First, we create the accept message, including the Client's address and a random nonce.
+This is done by simply calling `proposal.Accept()` on the proposal object.
+Then we send the accept message via the responder `responder.Accept()`.
+
+If this is successful, we call `startWatching()` to allow the Client to react to disputes on the accepted Channel automatically.
+Finally, we add the `PaymentChannel` to the Clients channel registry.
+
+.. code-block:: go
+
+        // Create a channel accept message and send it.
+        accept := lcp.Accept(
+            c.account,                // The account we use in the channel.
+            client.WithRandomNonce(), // Our share of the channel nonce.
+        )
+        ch, err := r.Accept(context.TODO(), accept)
+        if err != nil {
+            fmt.Printf("Error accepting channel proposal: %v\n", err)
+            return
+        }
+
+        // Start the on-chain event watcher. It automatically handles disputes.
+        c.startWatching(ch)
+
+        // Store channel.
+        c.channels <- newPaymentChannel(ch, c.currency)
+    }
+
+Handling Channel Updates
+~~~~~~~~~~~~~~~~~~~~~~~~
+For deciding how to handle incoming channel updates (off-chain!), we define `HandleUpdate()`.
+You can define logic here that decides under which conditions an update is accepted or rejected.
+In order to evaluate this, the current channel state `cur` given as `channel.State` and the proposed new state `next` given as `client.ChannelUpdate` come as arguments.
+For accepting or rejecting the proposed update `client.UpdateResponder` `r` is given.
+
+We will accept every update that increases our balance in our example. Therefore, we perform the following steps:
+
+#. Check if the type of asset is still the asset from the state before by calling `channel.AssetsAssertEqual()`.
```

Comment:
This will not be necessary anymore with the new release. Should we wait for the new release before publishing? not sure...